Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -33,4 +31,34 @@ def display_s
       name
     end
   end
+
+  private
+
+  def suitable_ruby
+    rubies.detect { |ruby| suitable?(ruby) }
+  end
+
+  def rubies
+    rubies = which_all("ruby")
+    if ruby_formula && ruby_formula.installed?
+      rubies.unshift Pathname.new(ruby_formula.bin/"ruby")
```

Comment:
`rubies <<` is our preferable style, thanks. Also `ruby_formula.bin/"ruby"` is already a `Pathname` so don't need `Pathname.new`